### DISCUSSION TITLE
[Q/A][Java SDK (Component)] Async call without retry

### DISCUSSION THREAD

**Discussion by eye-gu** [apache/dubbo#14889] [Question] [answered]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

When async calls, the provider timed out, but the consumer did not retry. Is the design like this, or did I use the wrong way?

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment, marked as answer]
This is not work as expected. But I'm thinking about how to solve this problem.

**Comment by AlbumenJ** [comment]
https://github.com/apache/dubbo/issues/14891
